### ISSUE TITLE
[polaris.shopify.com] Feedback (on /tools/polaris-migrator)

### ISSUE THREAD

**Issue by Sky-bits** [Shopify/polaris-react-archive#11010]
Need a polaris migration update with v12

**Comment by sam-b-rose** [comment]
Hi @Sky-bits! Thanks for catching this.

We currently have the [migrating from v11 to v12 page](https://polaris.shopify.com/version-guides/migrating-from-v11-to-v12), but I agree, this content should also be avaialble on the Polaris Migrator page.

I will get this added soon 👍

**Comment by Sky-bits** [comment]
@sam-b-rose I attempted to perform a migration of the Polaris version in our codebase to the latest version. However, I encountered a critical error during this process. Specifically, I received the following error message: "Error: No migration found for [component]." Despite this error, the associated component remains in the code repository, and the migration process is incomplete.

For example, I am ruining the this command 

`npx @shopify/polaris-migrator v12-react-avatar-component "**/*.{js,jsx,ts,tsx}" --force
Error: No migration found for v12-react-avatar-component

Expected Behavior:
I expected the migration process to execute without errors and successfully update the [component] to the latest Polaris version.

Actual Behavior:
The migration process failed with the error message mentioned above. Despite the error, the [component] is still present in the code repository, indicating that the migration was not successful.

`
Steps to reproduce:

1. Install migratio in same repo 'npm insta'
2. Navigate to the root directory.
3. Run the migration command with the specified component using the latest Polaris version.
4. Observe the error message: "Error: No migration found for [component]."

![Screenshot from 2023-10-25 13-24-40](https://github.com/Shopify/polaris/assets/110609906/85ac84ca-d1df-4a34-99f8-05477d93a887)

**Comment by sam-b-rose** [comment]
I think the missing migration issue is related to how `npx` caches the `@shopify/polaris-migrator` package.

Can you try running the following:

```sh
npx @shopify/polaris-migrator@latest v12-react-avatar-component "**/*.{js,jsx,ts,tsx}"
```

**Comment by sam-b-rose** [comment]
Original documentation-related issue should be closed by #11045.

@Sky-bits please let me know if you continue to have issues running the migration tool!
